### PR TITLE
Feature/session timeout

### PR DESCRIPTION
**Pull request by anthonyoleary**
Added a default timeout for session inactivity and a minimum password length of 8 characters.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -16,6 +16,10 @@ class User < ActiveRecord::Base
   has_many :granted_policies, class_name: "Policy", foreign_key: "granter_id"
 
   after_create :grant_implicit_policy
+  
+  def timeout_in
+    5.minutes
```
Comment (spalladino) [inline app/models/user.rb]: A timeout of 5 min can be a bit of a hassle, especially in development mode. I'd suggest moving this value to a setting in `settings.yml`, and keeping a much larger value in `development.yml`.

**Comment by nekron** [comment]
Would you mind making that change Anthony?

File: app/models/user.rb (hunk 1 above)
Comment (anthonyoleary) [inline app/models/user.rb]: Good point. I will do it.

Comment (spalladino) [inline app/models/user.rb]: If you need more info on how `Settings` are computed, we are using the [rails_config](https://github.com/railsconfig/config) gem for this.

**Comment by anthonyoleary** [comment]
I changed it to add different values for production and development settings. Hope it is ok now.

File: config/settings/development.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 nndd_url: "http://localhost:9000"
 location_service_url: "http://locations-stg.instedd.org"
+web_session_timeout: 120
```
Comment (nekron) [inline config/settings/development.yml]: Would you mind increasing this value to 3600, to simplify development and avoid having to log back in frequently?

If you want to develop and test the timeoutable behavior, you can always redefine it in a "development.local.yml" file that will be ignored by git.

**Comment by cisaacs01** [comment]
Can we reactivate the timeout feature please

**Comment by nditada** [comment]
It was never deactivated, it just has a long setting for development servers.

**Pull request merged**